src/Components/DrillIllustration.js

```diff
@@ -84,6 +85,14 @@ const DrillIllustration = props => {
     );
   };
 
+  const displayLine = () => {
+    if (props.drill.steps.length <= 1) {
+      return <View />;
+    } else {
+      return <View style={styles.lines} />;
```

Comment:
I think you could do something like `{props.drill.steps.length === 0 && <View style={styles.lines} />}` instead of extracting a function juste to have the condition

Reply:
I will discuss about this with Adélie. Thanks